ironfish-cli/src/commands/wallet/prune.ts

```diff
@@ -27,10 +31,52 @@ export default class PruneCommand extends IronfishCommand {
     CliUx.ux.action.start(`Opening node`)
     const node = await this.sdk.node()
     await NodeUtils.waitForOpen(node)
-    await node.wallet.open()
-    await node.wallet.walletDb.open()
     CliUx.ux.action.stop('Done.')
 
+    if (!node.chain.synced) {
+      this.log(
+        `Your node must be synced with the Iron Fish network to prune wallet transactions. Please try again later`,
+      )
+      this.exit(1)
+    }
```

Comment:
this will always be false since no node is running during a prune. We also don't need this since prune works on historical data and not forward looking data.